HousingRepairsOnlineApi/Properties/launchSettings.json

```diff
@@ -36,8 +36,11 @@
         "AZURE_STORAGE_CONNECTION_STRING": "<REPLACE WITH AZURE STORAGE CONNECTION STRING>",
         "STORAGE_CONTAINER_NAME": "housing-repairs-online",
         "GOV_NOTIFY_KEY" : "<REPLACE WITH GOV NOTIFY API URL>",
-        "CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY SMS TEMPLATE ID>",
-        "CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID": "<REPLACE WITH GOV NOTIFY EMAIL TEMPLATE ID>"
+        "CONFIRMATION_SMS_NOTIFY_TEMPLATE_ID": "8cb2913c-e724-48f5-8c3b-78854b1c086e",
+        "CONFIRMATION_EMAIL_NOTIFY_TEMPLATE_ID": "90e22bb6-5d1f-4909-b9e7-a3b040f2bd41",
+        "INTERNAL_EMAIL_NOTIFY_TEMPLATE_ID": "7f95924d-00c2-4345-baa1-ca015c326e25",
+        "INTERNAL_EMAIL": "<REPLACE WITH CSA INTERNAL EMAIL>",
+        "DAYS_UNTIL_IMAGE_EXPIRY": "<REPLACE WITH NUMBER OF DAYS BEFORE IMAGE SAS URI EXPIRES>"
```

Comment:
Still missing the number of days here

Reply:
I think similar to above, this will be different for different LA's